mlflow/utils/credentials.py

```diff
@@ -41,3 +46,129 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def login(backend="databricks"):
+    """Configure MLflow server authentication and connect MLflow to tracking server.
+
+    This method provides a simple way to connect MLflow to its tracking server. Currently only
+    Databricks tracking server is supported. Users will be prompted to enter the credentials if no
+    existing Databricks profile is found, and the credentials will be saved to `~/.databrickscfg`.
+
+    Args:
+        backend: string, the backend of the tracking server. Currently only "databricks" is
+            supported.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        mlflow.login()
+        with mlflow.start_run() as run:
+            mlflow.log_param("p", 0)
+    """
+    if backend == "databricks":
+        _databricks_login()
+    else:
+        raise ValueError(
+            f"Currently only 'databricks' backend is supported, received `backend={backend}`."
+        )
+
+
+def _check_databricks_auth():
+    # Check if databricks credentials are set.
+    from databricks.sdk import WorkspaceClient
+
+    try:
+        # If credentials are set correctly, `WorkspaceClient()` will not throw an error.
+        WorkspaceClient()
```

Comment:
Ran `WorkspaceClient()` with an invalid token on my machine. It doesn't fail.

Reply:
```python
>>> from databricks.sdk import WorkspaceClient
>>> w = WorkspaceClient()
>>> w.config.token
'foo'
>>> w.clusters.list()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/harutakakawamura/.pyenv/versions/miniconda3-4.7.12/envs/mlflow-dev-env/lib/python3.8/site-packages/databricks/sdk/service/compute.py", line 4070, in list
    json = self._api.do('GET', '/api/2.0/clusters/list', query=query)
  File "/Users/harutakakawamura/.pyenv/versions/miniconda3-4.7.12/envs/mlflow-dev-env/lib/python3.8/site-packages/databricks/sdk/core.py", line 942, in do
    raise self._make_nicer_error(message=message) from None
databricks.sdk.core.DatabricksError: Invalid access token.
```

Reply:
good call, fixed